client/src/languagePlugins/javascript/javascriptPlugin.ts

```diff
@@ -209,6 +214,32 @@ class JavascriptBlockKind implements Langs.Block {
   export const javascriptLanguagePlugin : Langs.LanguagePlugin = {
     language: "javascript",
     editor: javascriptEditor,
+    evaluate: (scopeDictionary:{}, node:Graph.Node) => {
+      let jsnode = <Graph.JsNode>node
+      let value = "yadda";
+      switch(jsnode.kind) {
+        case 'code': 
+          // console.log(JSON.stringify(jsnode));
+          let returnArgs = "{";
+          for (var e = 0; e < jsnode.exportedVariables.length; e++) {
+            returnArgs= returnArgs.concat(jsnode.exportedVariables[e]+":"+jsnode.exportedVariables[e]+",");
+          }
+          returnArgs = returnArgs.concat("}")
+          // console.log(returnArgs)
+          let evalCode = "function calc() {\n\t "+ jsnode.source +"\n\t return "+returnArgs+"\n}\n; calc()"
+          value = eval(evalCode);
+          break;
+        case 'export':
+          value = scopeDictionary[jsnode.variableName]
```

Comment:
Here, we should be able to access `jsnode.code` which should give us a `JsCodeNode` representing the value. The `value` of this node will then be the dictionary that we construct in the function passed to `eval` (the type of it is `any` so we may need to cast this to a dictionary).

Reply:
Would you please check out the new version?